### ISSUE TITLE
Search by Submission Class

### ISSUE THREAD

**Issue by timbmg** [kohler/hotcrp#370]
Is it possible to search by submission class? I.e. only return submissions that have been made to a certain submissions class?

**Comment by kohler** [comment]
Yes! Each submission class has a tag and you can search for that tag, `#CLASSNAME`. You can also search for `sclass:CLASSNAME`; that search will report a warning if CLASSNAME is not a known submission class.

**Comment by timbmg** [comment]
Thanks @kohler, that works great! I am using that, however, it seems that for PCs the search does not return their own submissions. Is that something that can be configured/changed?

**Comment by kohler** [comment]
The `sclass:NAME` search works for PC members’ own papers. It would be better if `#NAME` also worked, but for now I advise `sclass:NAME`.

**Comment by timbmg** [comment]
Oh my mistake. I was actually searching by #Name, however that didn't return submission if they had a conflict
